egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -108,6 +108,8 @@ nll_batch_size=100 # Affect GPU memory usage when computing nll
                    # during nbest rescoring
 k2_config=./conf/decode_asr_transformer_with_k2.yaml
 
+use_streaming=false # Whether to use streaming decoding
```

Comment:
Looks great, could you also update the help_message

Reply:
After you change this, I will merge the PR

Reply:
> 

Sorry, I'm not sure what you mean by "help_message". Does it mean my comment "# Whether to use streaming decoding"? How should I modify it?

Reply:
> 

Sorry, I'm not sure what you mean by "help_message". Does it mean my comment "# Whether to use streaming decoding"? How should I modify it? @ftshijt

Reply:
Just update the string defined here https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/asr.sh#L144 would be fine.

Reply:
> 

I see! thanks, I have updated it.